consensus/ibft/src/test/java/tech/pegasys/pantheon/consensus/ibft/ibftmessagedata/CommitPayloadTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.consensus.ibft.ibftmessagedata;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import tech.pegasys.pantheon.consensus.ibft.ConsensusRoundIdentifier;
+import tech.pegasys.pantheon.consensus.ibft.ibftmessage.IbftV2;
+import tech.pegasys.pantheon.crypto.SECP256K1.Signature;
+import tech.pegasys.pantheon.ethereum.core.Hash;
+import tech.pegasys.pantheon.ethereum.rlp.BytesValueRLPOutput;
+import tech.pegasys.pantheon.ethereum.rlp.RLP;
+import tech.pegasys.pantheon.ethereum.rlp.RLPInput;
+
+import java.math.BigInteger;
+
+import org.junit.Test;
+
+public class CommitPayloadTest {
+  private static final ConsensusRoundIdentifier ROUND_IDENTIFIER =
+      new ConsensusRoundIdentifier(0x1234567890ABCDEFL, 0xFEDCBA98);
+
+  @Test
+  public void roundTripRlp() {
+    final Signature signature = Signature.create(BigInteger.ONE, BigInteger.TEN, (byte) 0);
+    final Hash hash = Hash.fromHexStringLenient("0x8523ba6e7c5f59ae87");
+
+    final CommitPayload expectedCommitPayload =
+        new CommitPayload(ROUND_IDENTIFIER, hash, signature);
+    final BytesValueRLPOutput rlpOut = new BytesValueRLPOutput();
+    expectedCommitPayload.writeTo(rlpOut);
+
+    final RLPInput rlpInput = RLP.input(rlpOut.encoded());
+    final CommitPayload commitPayload = CommitPayload.readFrom(rlpInput);
+    assertThat(commitPayload.getRoundIdentifier()).isEqualTo(ROUND_IDENTIFIER);
```

Comment:
nit: Do we want to check against "initial values" or just against "expectedCommitPayload"? If the latter, look at using "isEqualToComparingFieldByField".
(probably still need to do the MessageType")
I am assuming we trust that the constructor & getters do the right thing.